### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by anderCM**
- Create a website consisting of 3 pages: `Home`, `Calculator`, and `Quote`.
- Add your own styles to improve the look and feel.
- Use React basic styling (inline styles, importing CSS and pre-processed CSS files).
- For every page, you should create a <Route>.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router } from 'react-router-dom';
+import reportWebVitals from './reportWebVitals';
```
Comment (VICTOR-CHUKWU) [inline src/index.js:5]: - [x] [Optional] remove this line as it is part of react boiler plate generated with npx create react-app, also clean up your index.js file and remove commented codes

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+const App = () => (
+  <>
+    <NavBar />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/calculator" element={<Calculator />} />
+      <Route path="/quote" element={<Quote />} />
+      <Route path="*" element={<ErrorRoute />} />
+    </Routes>
+  </>
```
Comment (VICTOR-CHUKWU) [inline src/App.js:19]: - [x] good use of react router :clap:

File: src/Components/ErrorRoute.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+const ErrorRoute = () => (
+  <>
+    <Container className="text-center">
+      <h1 className="text-center fw-bold text-white my-3">Page not found</h1>
+      <img src={NotFound} alt="Page not Found" className="w-25 h-25 my-3" />
+    </Container>
+  </>
+);
+
+export default ErrorRoute;
```
Comment (VICTOR-CHUKWU) [inline src/Components/ErrorRoute.js:15]: - [x] so thoughtful of you to add the error page :+1:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @anderCM , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] All linters passed
- [x] good commit history
- [x] good use of react router

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (anderCM) [reply src/App.js:19]: Thanks! I appreciate your comment!

File: src/Components/ErrorRoute.js (hunk 3 above)
Comment (anderCM) [reply src/Components/ErrorRoute.js:15]: Thanks for your comment 😊

**Pull request merged**